### PR TITLE
New Blog Content

### PR DESCRIPTION
**Pull request by ehsanmasdar**

### CODE REVIEW

File: _posts/2018-03-20-infra.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+categories: blog posts		
+author: Ehsan Asdar
+image: infrat.png
+---
+[HackGT](https://hack.gt) is a student organization at Georgia Tech that runs a series of events every year that empower students to learn and hone their skills in Computer Science. Our marquee event (of the same name, HackGT) takes place each fall, hosting over 1000 hackers at Georgia Tech for 36 hours of learning and creating, while other events such as [Catalyst](http://catalyst.hack.gt) and [HackGTeeny](http://teeny.hack.gt) strive to show students how intriguing and rewarding careers in STEM, and specifically CS, can be.
+{% render_image  infra1.gif%}
+
+The HackGT tech team builds software to support our event organizing teams. Our variety of responsibilities range from developing educational content (workshops, tech talks, CTFs) to writing applications to manage hacker data. This is the first in a series of posts about HackGT and will focus on how we built our infrastructure to ensure a clean, consistent development-release cycle, complete with zero-downtime deployment, monitoring and alerting, and effortless metrics collection.
+
+# Motivation
```
Comment (bunsenmcdubbs) [inline _posts/2018-03-20-infra.md]: This should probably be `##` (h2). The blog post title is already `h1` so semantically all the headers inside the post should be one level below that.

**Review by bunsenmcdubbs — CHANGES_REQUESTED** [review]

**Review by bunsenmcdubbs — APPROVED** [review]

**Review by bunsenmcdubbs — COMMENTED** [review]
I think we should change the date of the blog posts to when it is actually published (not March lol)

**Comment by kexin-zhang** [comment]
The footer looks like this for me... 

image

**Review by kexin-zhang — COMMENTED** [review]
Alright, so Jekyll was rendering the page weirdly because of how we were truncating the preview for the blog post, but it's fixed now. I think we're good to merge this once we get an image for the postgres post and fix the date of the other post.

**Review by kexin-zhang — APPROVED** [review]

**Pull request merged**